src/simtools/schemas/model_parameters/dsum_threshold.schema.yml

```diff
@@ -28,7 +28,6 @@ instrument:
   type:
     - MSTN
     - MSTS
-    - SSTS
```

Comment:
Why did you remove only SSTs if it is relevant only for FlashCams?

Reply:
Same, see above.